Libraries/Text/Text/RCTTextView.m

```diff
@@ -423,34 +459,28 @@ - (BOOL)becomeFirstResponder
 
 - (BOOL)resignFirstResponder
 {
-  if (![super resignFirstResponder]) {
+  // Don't relinquish first responder while selecting text.
+  if (_selectable && NSRunLoop.currentRunLoop.currentMode == NSEventTrackingRunLoopMode) {
     return NO;
   }
 
-  // If we've lost focus, notify listeners
-  [_eventDispatcher sendEvent:[RCTFocusChangeEvent blurEventWithReactTag:self.reactTag]];
-
-  return YES;
+  return [super resignFirstResponder];
 }
 
-#endif // ]TODO(macOS GH#774)
-
-- (BOOL)canBecomeFirstResponder
+- (BOOL)canBecomeKeyView
 {
+  // RCTText should not get any keyboard focus unless its `selectable` prop is true
```

Comment:
`RCTTextView`?

Reply:
Fixed in b72d4bcaf5e482feeb784494cb2ab069a0aa26fe